### ISSUE TITLE
SVG looks alias in IE11, Windows 7

### ISSUE THREAD

**Issue by geoffyuen** [BoomTownROI/boomsvgloader#1]
I don't think it's an issue with your code but ajaxing introduces more jagged edges in the rendered SVG. With certain SVGs areas can look like their being filled in. 

If you simply embed the SVG in html the edges are smooth.

**Comment by mcraiganthony** [comment]
@geoffyuen Any chance you can share a screen grab showing the ajaxed and non-ajaxed versions?

**Comment by geoffyuen** [comment]
http://codepen.io/geoffyuen/pen/WQQpoz

This example has a spritesheet directly embedded and another spritesheet ajaxed in (using different ids for each symbol but otherwise identical). There is also a screenshot at the bottom taken in a vm.

**Comment by mcraiganthony** [comment]
@geoffyuen Thanks for putting that together. I can confirm that I'm seeing the same results via Browserstack.

**Comment by mcraiganthony** [comment]
@geoffyuen This issue also occurs in Win 7 IE10.

I found the problem. Apparently IE doesn't like "fill-rule:evenodd" in the SVG.

http://codepen.io/mcraiganthony/pen/JYGbgO

In the above pen, I used your icons and some of my icons. I ripped out all instances of "fill-rule" from your sprite. Now the rendered SVGs look identical.

**Comment by geoffyuen** [comment]
@mcraiganthony Fantastic find! I'll have to figure out a way to check for this fill-rule in my gulpfile which generates my spritesheet.

**Comment by geoffyuen** [comment]
I may have spoken too soon. I'm looking at your example in a VM and it still looks the same (bold, aliased) to me. Maybe it's an issue with my VM. I'll take a closer look tonight.

**Comment by mcraiganthony** [comment]
This is what I'm seeing in my VM Windows 7 / IE10. Pulling the fill-rules messed up a couple of your icons but you can still see that they are identical.

<img width="236" alt="screen" src="https://cloud.githubusercontent.com/assets/1093578/9856816/e5cff9fc-5ae4-11e5-91c4-48ec88434e80.png">

**Comment by mcraiganthony** [comment]
@geoffyuen I just viewed it in Win 7 / IE11 and you example does some heavier lines but it's much improved. I'm guessing that it has more to do with the actual SVG source than the SVG sprite build task. Creating SVGs can be a bit finicky.

**Comment by geoffyuen** [comment]
@mcraiganthony , yeah it looks ok in ie10 but not in ie11. In the screenshot I have IE10 on the left and ie11 on the right:

![screen shot 2015-09-14 at sep 14 15 - 12 48 21 pm](https://cloud.githubusercontent.com/assets/1411586/9857235/05ef981a-5adf-11e5-996b-54143d0f43ae.png)

**Comment by geoffyuen** [comment]
I guess it's just going to have to be one of those "iethings". Unfortunately (fortunately?), people are starting to use this ajaxing technique quite a bit but at least Edge is ok.

**Comment by mcraiganthony** [comment]
I see that my icons are also a bit scrawzed. Damn this is going to bother me. It's crazy that IE10 is fixed and IE11 is 'improved' but not completely fixed.
